### DISCUSSION TITLE
How big is yolo v5 p6?

### DISCUSSION THREAD

**Discussion by hamzahmhmmd** [ultralytics/yolov5#3194] [Q&A] [answered]
Anyone know how big is yolo v5 p6 in MB, especially from [YOLOv5s6 ](https://github.com/ultralytics/yolov5/blob/master/models/hub/yolov5s6.yaml) and [yolov5s-transformer.yaml](https://github.com/ultralytics/yolov5/blob/master/models/hub/yolov5s-transformer.yaml) ?

I just find the size of P5 ![I just find the size of P5](https://user-images.githubusercontent.com/26833433/103595982-ab986000-4eb1-11eb-8c57-4726261b0a88.png)

Thanks in advance

**Comment by glenn-jocher** [comment, marked as answer]
@hamzahmhmmd you can see the file sizes of the current YOLOv5 models in the latest release assets:
https://github.com/ultralytics/yolov5/releases/tag/v5.0

<img width="683" alt="Screenshot 2021-05-17 at 13 23 40" src="https://user-images.githubusercontent.com/26833433/118480890-218bbe00-b713-11eb-9013-5df7a29fb3f7.png">

**Reply by hamzahmhmmd** [reply]
Thanks alot glenn
